analysis/report_templates/default.html

```diff
@@ -153,6 +155,7 @@ <h5 class="center-align">Mann-Whitney U test</h4>
             {% for benchmark in experiment.benchmarks %}
             <li><a href="#{{ benchmark.name }}">{{ benchmark.name }}</a></li>
             {% endfor %}
+            <li><a href="#data">experiment data</a></li>
```

Comment:
consider something like "raw data download"?

Reply:
feels simple is better, also matches style "experiment summary"